luminance/src/blending.rs

```diff
@@ -76,3 +76,27 @@ pub struct Blending {
   /// Destination factor.
   pub dst: Factor,
 }
+
+/// Blending configuration to represent combined or separate options.
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum BlendOption {
+  /// blending with combined RGBA
```

Comment:
Please use upper case as first letter and dot at the end of documentation lines.